src/utils/index.ts

```diff
@@ -550,3 +550,20 @@ const auth = {
       : user.subscribedCourses.includes(courseId) ||
         auth.isCentreManager(user, centreId),
 };
+
+export const contentPriceValidator = (
+  price: number,
+  currency: string,
+  supportedCurrencies: Record<string, any>
+) => {
+  if (!price) return { price: 0, currency: "" };
+  if (price > 0 && !currency) throw new ValidationError("currency is required");
+  if (price) {
```

Comment:
This is not longer needed! you've already checked if(!price) on line 559

Reply:
True